docker-compose.yaml

```
@@ -1,6 +1,150 @@
+            start_period: 3s
+            timeout: 10s
+        ports:
+            - "3000:3000"
+        volumes:
+            - ./zoo-grafana/grafana.ini:/etc/grafana/grafana.ini
+            - grafana-data:/var/lib/grafana
+        networks:
+            - zoo
+        environment:
+            - GF_SECURITY_ADMIN_PASSWORD=very-secret
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)